square/builder.go

```diff
@@ -128,8 +134,8 @@ func (b *Builder) Export() (Square, error) {
 	// of blobs within a namespace because b.Blobs are already ordered by tx
 	// priority.
 	sort.SliceStable(b.Blobs, func(i, j int) bool {
-		ns1 := append([]byte{byte(b.Blobs[i].Blob.NamespaceVersion)}, b.Blobs[i].Blob.NamespaceId...)
-		ns2 := append([]byte{byte(b.Blobs[j].Blob.NamespaceVersion)}, b.Blobs[j].Blob.NamespaceId...)
+		ns1 := b.Blobs[i].Blob.Namespace().Bytes()
+		ns2 := b.Blobs[j].Blob.Namespace().Bytes()
 		return bytes.Compare(ns1, ns2) < 0
```

Comment:
Looks like this can be a `ns1.Compare(ns2) < 0` (https://github.com/celestiaorg/go-square/blob/main/namespace/namespace.go#L195)